Wireframe/index.html

```diff
@@ -1,13 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
   </head>
+
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+       <main>
+      <header class="header-container">
+        <h1>Welcome to Git</h1>
+        <p>
+          A distributed version control system, where every developer gets their
+          local repository with full commit history.
+        </p>
+      </header>
+
+      <section class="article1">
+        <img class="article1_img" src="/Wireframe/git2.png">
```

Comment:
img element must have an alt attribute, except under certain conditions. 
```suggestion
<img class="article1_img" src="/Wireframe/git2.png" alt"adds image description here">
```

More about the img element here: https://www.w3.org/WAI/tutorials/images/